MerlinAU.sh

```diff
@@ -1135,14 +1135,13 @@ check_memory_and_prompt_reboot() {
         availableRAM_kb=$(_GetAvailableRAM_KB_)
         if [ "$availableRAM_kb" -lt "$required_space_kb" ]; then
             # In an interactive shell session, ask user to confirm reboot #
-            if [ "$inMenuMode" = true ]; then
```

Comment:
The reason to use the "$isInteractive" variable along with the prompt function on the same line is that we need to check for **both** conditions as a logical **AND** boolean so that if either one is false we fall back on the "**else**" statement to call the "cleanup & exit" procedures. The way it was before, if the user answers NO, there's no code to do the cleanup & exit.